PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+## Description
+<!-- Expectations for PRs: [RFC66 - Pull Request & Code Review Standards](https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards) -->
+
+<!-- Have you included a short summary of what the PR does, a description of the problem being solved and the pros/cons of the approach? -->
+
+<!-- Does it make sense for this PR to be of this size? If the PR is large, consider breaking it down into smaller incremental PRs. -->
```

Comment:
I think a brief checklist could be useful to immediately highlight to reviewers whether the author has considered the PR expectations.
```suggestion
## Checklist
<!-- Does it make sense for this PR to be of this size? If the PR is large, consider breaking it down into smaller incremental PRs. -->
[ ] Is this PR a reasonable size?
```